### PR TITLE
Separate routers

### PR DESCRIPTION
**Pull request by tisnik**
## Description

- Separate routers handling functions
- It is hard/impossible to call any code from `main.py` from unit or integration tests, so it's better to refactor all functions out of it

## Type of change

- [x] Refactor
- [ ] New feature
- [ ] Bug fix
- [ ] CVE fix
- [ ] Optimization
- [ ] Documentation Update
- [ ] Configuration Update
- [ ] Bump-up dependent library

### CODE REVIEW

File: ols/app/routers.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+"""REST API routers utility functions."""
```
Comment (onmete) [inline ols/app/routers.py:1]: Just "REST API routers"?

**Comment by onmete** [comment]
one nit

/lgtm

File: ols/app/routers.py (hunk 1 above)
Comment (tisnik) [reply ols/app/routers.py]: ok, changed @onmete

**Pull request merged**